web/src/site-admin/backend.tsx

```diff
@@ -168,6 +168,42 @@ export function fetchAllRepositoriesAndPollIfEmptyOrAnyCloning(
     )
 }
 
+interface RepositoriesPermissionsArgs {}
+
+export function fetchAllRepositoriesPermissions(
+    args: RepositoriesPermissionsArgs
+): Observable<GQL.IRepositoryPermissionsConnection> {
+    args = {
+        descending: true,
+        ...args,
+    } // apply defaults
+    return queryGraphQL(
+        gql`
+            query RepositoriesPermissions($first: Int, $query: String, $descending: Boolean) {
+                repositoriesPermissions(first: $first, query: $query, descending: $descending) {
+                    nodes {
+                        repository {
+                            name
+                            url
+                        }
+                        permissions {
+                            updatedAt
+                        }
+                    }
+                    totalCount
+                    pageInfo {
+                        hasNextPage
+                    }
+                }
+            }
+        `,
+        args
```

Comment:
In order to not alter the input, I would just inline it 

```suggestion
    return queryGraphQL(
        gql`
            query RepositoriesPermissions($first: Int, $query: String, $descending: Boolean) {
                repositoriesPermissions(first: $first, query: $query, descending: $descending) {
                    nodes {
                        repository {
                            name
                            url
                        }
                        permissions {
                            updatedAt
                        }
                    }
                    totalCount
                    pageInfo {
                        hasNextPage
                    }
                }
            }
        `,
        {
	        descending: true,
	        ...args,
	    }
```